pipeline/metadata/optics/xmm.rml

```diff
@@ -0,0 +1,30 @@
+<xml>
+	<globals>
+		<constant name="R0" value="153.126" />
+		<constant name="pi" value="3.1415927" />
+	</globals>
+
+	<!-- Turn verbose level to "debug" to see additional output -->
+    	<!-- All values in mm; innermost to outermost mirror -->
+	<TRestAxionOptics name="basic" verboseLevel="info" >
+		<parameter name="lMirr" value="300mm" />
+		<parameter name="R1" value="153.126, 155.419, 157.731, 160.065, 162.428, 164.812, 167.225, 169.66, 172.124, 174.608, 177.123, 179.658, 182.224, 184.971, 187.749, 190.556, 193.394, 196.263, 199.161, 202.09, 205.05, 208.09, 211.16, 214.261, 217.392, 220.553, 223.755, 226.987, 230.249, 233.552, 236.885, 240.248, 243.652, 247.298, 250.984, 254.711, 258.478, 262.276, 266.114, 269.992, 273.911, 277.87, 281.869, 285.92, 290.01, 294.292, 298.676, 303.109, 307.584, 312.108, 316.674, 321.289, 325.955, 330.672, 335.439, 340.246, 345.104, 350.013" />

```

Comment:
I thing it is interesting to have REST automatically parsing those values to `vector<double>`?

Reply:
Yes, that would be very interesting! we are using this kind of parameters in few places.

Reply:
We read them using `StringToElements`, but perhaps we should have a template that works for `int`, `float` and `double`?